### PR TITLE
Feat: OPX3.1-dev1

### PR DESCRIPTION
**Pull request by GarrickHe**
* Feature: VxLAN Support
* Feature: SAI 1.2
* Update: Add IPv6 Extended prefix(>/64) route entry size
* Update: SAI-API Attributes for supporting split horizon
* Update: additional ip stat attributes
* Update: Display TCAM utilization
* Update: Add support for match brige type in ACL
* Update: Add egress filter attribute to bridgeport
* Bugfix: Fix the VLAN Id attribute

Signed-off-by: Garrick He <garrick_he@dell.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,17 @@
+opx-sai-api (36.0.0) unstable; urgency=medium
+
+  * Feature: VxLAN Support
```
Comment (jeffyin-dell) [inline debian/changelog]: Should we clarify that this is Static L2 VxLAN support, or is all VxLAN support being added here to SAI?

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+  * Feature: SAI 1.2
+  * Update: Add IPv6 Extended prefix(>/64) route entry size
+  * Update: SAI-API Attributes for supporting split horizon
+  * Update: additional ip stat attributes
```
Comment (jeffyin-dell) [inline debian/changelog]: Capitalize "Additional"

File: debian/changelog (hunk 1 above)
Comment (GarrickHe) [reply debian/changelog]: I took this from the feature/VxLan branch, will have to ask Atanu. I assumed all his commits into that branch were vetted.

File: experimental/saiaclextensions.h [hunk 3]
```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014 Microsoft Open Technologies, Inc.
```
Comment (atanu-mandal) [inline experimental/saiaclextensions.h:2]: Do we want to update the copyright ?

File: experimental/saiextensionaclslice.h [hunk 4]
```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2014 Microsoft Open Technologies, Inc.
```
Comment (atanu-mandal) [inline experimental/saiextensionaclslice.h:2]: Do we want to update the copyright ?

File: inc/sai.h [hunk 5]
```diff
@@ -15,7 +15,7 @@
  *
  *    Microsoft would like to thank the following companies for their review and
  *    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
- *    Dell EMC, Facebook Inc., Marvell International Ltd.
+ *    Dell EMC, Facebook, Inc., Marvell International Ltd.
```
Comment (atanu-mandal) [inline inc/sai.h]: Earlier one was ok I guess.

File: inc/saiacl.h [hunk 6]
```diff
@@ -15,7 +15,7 @@
  *
  *    Microsoft would like to thank the following companies for their review and
  *    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
- *    Dell EMC, Facebook Inc., Marvell International Ltd.
+ *    Dell EMC, Facebook, Inc., Marvell International Ltd.
```
Comment (atanu-mandal) [inline inc/saiacl.h]: Earlier one was ok I guess.

File: inc/saibuffer.h [hunk 7]
```diff
@@ -15,7 +15,7 @@
  *
  *    Microsoft would like to thank the following companies for their review and
  *    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
- *    Dell EMC, Facebook Inc., Marvell International Ltd.
+ *    Dell EMC, Facebook, Inc., Marvell International Ltd.
```
Comment (atanu-mandal) [inline inc/saibuffer.h]: Earlier one was ok I guess.

File: debian/changelog (hunk 1 above)
Comment (atanu-mandal) [reply debian/changelog]: This should have all VxLAN support.

Comment (atanu-mandal) [reply debian/changelog]: This should have all VxLAN support.

**Comment by GarrickHe** [comment]
Pull-request updated, HEAD is now 481f854aaf847657fe9c55431a60ead90a4258ed

**Review by atanu-mandal — APPROVED** [review]

**Pull request merged**